assets/js/blocks/customer-account/style.scss

```
@@ -1,4 +1,8 @@
 .wp-block-woocommerce-customer-account {
+	&.aligncenter > a {
+		justify-content: center;
+	}
```

If you accept the solution that I posted above, we can also change this css:

```suggestion
	&.aligncenter {
		width: fit-content; // or max-content
	}
```